package.json

```diff
@@ -91,7 +91,7 @@
     "core-js": "3.6.5",
     "flexboxgrid": "6.3.1",
     "focus-trap": "5.1.0",
-    "fundamental-styles": "0.10.0",
+    "fundamental-styles": "0.11.0",
```

Comment:
shouldn't package-lock also be committed?

Reply:
@kavya-b This and other scss imports are temporary until we introduce `Fundamental Styles in v.11` #2882. @manu-kr will update the PR after merge.

Reply:
Oh, I see, wasn't aware.